src/app/components/close-button/index.tsx

```diff
@@ -4,16 +4,21 @@ import { CloseIcon } from "../icons";
 type CloseButtonProps = {
   onClick: () => void;
   label?: string;
+  className?: string;
 };
 
-export const CloseButton = ({ onClick, label }: CloseButtonProps) => {
+export const CloseButton = ({
+  onClick,
+  label,
+  className,
```

Comment:
since className is optional, set a default value.

className = '',

This will ensure the embedded value for this:
className={`flex h-fit w-fit items-center justify-center ${className}`}

will not result in "flex h-fit w-fit items-center justify-center **undefined**"

It's not breaking, just ugly :D